docs/cli/edgedb.rst

```diff
@@ -129,3 +129,89 @@ Options
     :cli:synopsis:`<timeout>` value must be given using time units
     (e.g. ``hr``, ``min``, ``sec``, ``ms``, etc.). The default
     value is ``10s``.
+
+
+Backslash Commands
+==================
+
+.. rubric:: Introspection
+
+The introspection commands share a few common options that are available to
+many of the commands:
+
+- ``-v``- Verbose
+- ``-s``- Show system objects
+- ``-c``- Case-sensitive pattern matching
+
+:cli:synopsis:`\\d [-v] NAME`
```

Comment:
`\d` is currently a shortcut for `\describe` And we have `\describe object NAME` and `\describe schema`